x-pack/legacy/plugins/reporting/server/usage/types.d.ts

```diff
@@ -51,9 +73,22 @@ export type JobTypes = { [K in BaseJobTypeKeys]: AvailableTotal } & {
   };
 };
 
+interface StatusCounts {
+  [statusType: string]: number;
+}
+
+interface StatusByAppCounts {
+  [statusType: string]: {
+    [appType: string]: {
+      [jobType: string]: number;
```

Comment:
If my above understanding is correct, I think you'll want to swap appType (maybe rename to appName for consistency?) and jobType here